### PR TITLE
Evmos mainnet wBTC/renBTC base pool and tBTC metapool integration

### PR DESCRIPTION
**Pull request by ug02fast**
image

### CODE REVIEW

**Comment by penandlim** [comment]
Tested deposits via wrapped and unwrapped. LGTM

**Review by penandlim — APPROVED** [review]

File: src/constants/index.ts [hunk 1]
```diff
@@ -1032,6 +1083,16 @@ export const POOLS_MAP: PoolsMap = {
     route: "btcv2",
     rewardPids: buildPids({ [ChainId.MAINNET]: 4, [ChainId.HARDHAT]: 4 }),
   },
+  [EVMOS_BTC_POOL_NAME]: {
+    name: EVMOS_BTC_POOL_NAME,
+    addresses: EVMOS_BTC_SWAP_ADDRESSES,
+    lpToken: EVMOS_BTC_SWAP_TOKEN,
+    poolTokens: EVMOS_BTC_POOL_TOKENS,
+    isSynthetic: true,
```
Comment (hammeiam) [inline src/constants/index.ts:1091]: false

File: src/constants/index.ts [hunk 2]
```diff
@@ -1246,6 +1307,19 @@ export const POOLS_MAP: PoolsMap = {
     route: "evmosusd",
     rewardPids: buildPids({}),
   },
+  [TBTC_EVMOS_BTC_METAPOOL_NAME]: {
+    name: TBTC_EVMOS_BTC_METAPOOL_NAME,
+    addresses: TBTC_EVMOS_BTC_META_SWAP_DEPOSIT_ADDRESSES,
+    lpToken: TBTC_EVMOS_BTC_SWAP_TOKEN,
+    poolTokens: TBTC_EVMOS_POOL_TOKENS,
+    isSynthetic: true,
```
Comment (hammeiam) [inline src/constants/index.ts:1315]: false

**Review by hammeiam — APPROVED** [review]
change isSynthetic property. lgtm otherwise

**Pull request merged**